### ISSUE TITLE
Create new IHP task for POA updates

### ISSUE THREAD

**Issue by araposo-tistatech** [department-of-veterans-affairs/caseflow#15023]
## User or job story
Job story: When a POA is updated in VBMS in some cases a new IHP task needs to be created and sent to the POA even if an IHP task has been completed on the case.

If the POA is a State Representative a new IHP is not needed. 
If the POA is a Private Attorney, VLJs will send a letter to the new POA giving them the opportunity to provide arguments on the appeal if they wish. They have approximately 37 days. (IHP is not needed) 
If the POA is a Board based VSO, then the new POA gets the opportunity to prepare an IHP. The previous IHP becomes moot. 

## Acceptance criteria
- [ ] Please put this work behind the feature toggle: IHP_Notification
- [ ] New IHP task is created if POA is Board based VSO

## Release notes

If a POA is updated to a Board based VSO Caseflow will now automatically create a new IHP task for completion by the new VSO.  

### Out of scope

Creating a new task for VLJs if the new POA is a private attorney (letter sent to the new POA to provide arguments.)

**Comment by araposo-tistatech** [comment]
@mbaenatan @RyanRoyUX now that we received more clarity on this story do we think we need to notify the attorney working on the case that a new IHP task was created due to a POA change. For instance display a banner on the case noting an IHP task was created or maybe adding text next to the active tasks saying something like "New".

**Comment by hschallhorn** [comment]
As detailed in #14889, we _should_ already have a job that does this, but it may not be working as intended.

**Comment by hschallhorn** [comment]
@araposo-tistatech What is this difference between #14975 and this ticket?

**Comment by araposo-tistatech** [comment]
@hschallhorn in #14975 the IHP task is still active and needs to be cancelled. In this ticket the IHP task has already been completed. I wasn't sure if those two scenarios should be combined.

**Comment by ryanroyroy** [comment]
I think it will be clear and we could test that of course but also we could **bold the new tasks** as well

**Comment by mbaenatan** [comment]
Per discussion on 8/24 we will not move forward with additional designs at the moment. Attorneys will not have an actions drop down menu and will be able to see the case is blocked by an IHP task on the case details page.
